wsd/README.md

```diff
@@ -1,260 +0,0 @@
-
-HTTPS is the default. HTTP-only mode can be enabled with --disable-ssl
-configure option.
-
-If you plan to hack on coolwsd, you probably want to familiarize
-yourself with coolwsd's --o:num_prespawn_children switch, and the 'connect'
-test program.
-
-For interactive testing, you can use the 'connect' program. It accepts
-"commands" from the protocol on standard input.
-
-## Test running with integration for developers
-
-Unless you want to test SSL itself, it is easier to go for the non-SSL option.
-
-Setup Nextcloud or ownCloud on localhost, and install the richdocuments app.
-Good tutorials exist how to install ownCloud or Nextcloud, we don't repeat
-them here. richdocuments is called Collabora Online in the respective app
-stores / marketplaces / whatever.
-
-When you have a running Nextcloud or ownCloud instance at
-http://localhost/nextcloud or at http://localhost/owncloud
-go to Collabora Online settings, and set the WOPI URL to
-http://localhost:9980
-
-Then in the build tree, edit the generated coolwsd.xml and set ssl setting to
-false. You can run make run, and test coolwsd with the ownCloud or Nextcloud
-integration.
-
-Note: if SSL is enabled in either Online or the integration, both must
-have SSL enabled. That is, you must access NC/OC using https:// as well
-as configure the Collabora Online endpoint in NC/OC as https://localhost:9980.
-
-## Admin Panel
-
-You can access the admin panel by directly accessing the admin.html file
-from browser directory. See browser/README for more details.
-
-Websocket connections to admin console can be made at path: /adminws/ on the
-same url and port as coolwsd is running on. However, one needs a JWT token to
-authenticate to the admin console websocket. This is stored as a cookie with
-`Path: /adminws/` when user successfully authenticates when trying to access
-/browser/dist/admin/admin\*html files (HTTP Basic authentication). Token
-is expired after every half an hour, so websocket connection to admin console
-must be established within this period.
-
-It should also be possible to do various sorts of tasks such as killing
-documents that are open for more than 10 hours etc. See protocol.txt for
-various commands. Only tricky thing here is getting the JWT token which can
-be obtained as described above.
-
-## Debugging
-
-When debugging, you want to pass `--o:num_prespawn_children=1` to coolwsd
-to limit the number of concurrently running processes.
-
-When a crash happens too early, you also want to
-
-    export SLEEPFORDEBUGGER=<number of seconds>
-
-or
-    export PAUSEFORDEBUGGER=1
-
-so that you have time to attach to the process.
-
-Then run coolwsd, and attach your debugger to the process you are
-interested in. Note that simply attaching gdb via `gdb -p` is not meant to work, your options are:
-
-- `sudo gdb -p <PID>`, which is easy to remember or
-
-- `gdb -iex "set sysroot /" -p <PID>`, which can run as an unprivileged user, since we switched from
```

Comment:
I consider this part highly non-trivial, any reason you dropped the instructions on how to attach gdb to a jailed kit process? (I mean primarily the 'set sysroot' part and the matching alias.) Thanks.

Reply:
see https://github.com/CollaboraOnline/CollaboraOnline.github.io/pull/140

Reply:
Ah, sorry, I didn't see the above review comment. Makes sense then, thanks.